tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/pullquote_block.html

```diff
@@ -1,26 +1,51 @@
+            <g style="mix-blend-mode:multiply">
+                <path d="M8.59 6.934c.1 1.181 1.73 1.554 2.406.558 3.973-5.98 9.391-11.01 13.069-12.448l-.032.66c-1.405 42.635 30.37 67.332 41.634 74.642 1.065.713 2.448-.322 1.953-1.449-6.89-15.108-3.408-28.904 9.999-41.85 2.545-2.375 5.36-5.472 7.785-8.588 0 0 2.918-3.607 5.746-8.615l.15.155c4.888 7.734 5.635 16.737 5.636 21.662.022 1.177 1.566 1.693 2.313.847 3.491-4.093 9.539-12.472 10.852-21.89 3.126-22.78-10.24-46.876-34.836-54.319-1.16-.35-2.077.781-1.418 1.769 2.038 3.333 3.152 7.944 2.872 13.81C76.072-14.556 59.433 5.29 59.433 5.29S44.76-16.077 45.407-29.643c.283-5.94 1.83-10.422 4.177-13.542a1.205 1.205 0 00-1.244-1.898C23.872-40.392 11.524-20.045 9.19-7.296c-1.052 5.609-.83 10.838-.6 14.23z" fill="var(--color--accent-two)" style="mix-blend-mode:screen"/>
+            </g>
+            <g style="mix-blend-mode:multiply">
+                <path d="M72.805 30.882c-.374.779-1.505.626-1.67-.22-.945-5.06-3.055-9.834-5.006-11.73l-.154.457c-10.318 29.366-36.94 38.252-45.996 40.418-.861.22-1.465-.83-.855-1.473 8.336-8.564 9.757-18.821 4.668-30.974-.988-2.251-1.957-5.061-2.673-7.786 0 0-.9-3.182-1.376-7.296l-.135.068c-5.13 4.045-7.97 9.986-9.265 13.338-.324.795-1.438.761-1.688 0C7.52 22.027 5.89 14.817 7.535 8.08c4.01-16.282 18.81-29.349 36.347-28.282.827.051 1.11 1.05.434 1.558-2.168 1.76-4.085 4.62-5.45 8.682-3.157 9.394 2.166 27.048 2.166 27.048S55.943 6.203 59.1-3.19c1.382-4.112 1.58-7.548.914-10.257-.183-.795.647-1.421 1.287-.981C75.567-5.136 78.039 11.79 76.166 21.048c-.808 4.08-2.323 7.583-3.36 9.834z" fill="var(--color--accent-three)" style="mix-blend-mode:screen"/>
+            </g>
+        </g>
     </svg>
-    <p>{{ value.quote }}</p>
-    <cite>
-        {% if value.attribution %}
-            <span>{{ value.attribution }}</span>
-        {% endif %}
 
-        {% if value.name %}
-            <span>{{ value.name }}</span>
-        {% endif %}
+    <p class="blockquote__text heading heading--four">{{ value.quote }}</p>
+
+    <div class="blockquote__reference">
+        <div class="blockquote__meta">
+            {% if value.author_image %}
+                {% include "patterns/atoms/avatar/avatar.html" with avatar=value.author_image classes="avatar--small" %}
+            {% elif value.company_logo %}
+                {% image value.company_logo width-110 format-webp class="blockquote__company-logo" %}
+            {% endif %}
+
+            <cite class="blockquote__cite">
+                {% if value.name %}
+                    <span class="blockquote__name">{{ value.name }}</span>
+                {% endif %}
+
+                {% if value.role %}
+                    <span class="blockquote__role">{{ value.role }},</span>
+                {% endif %}
 
-        {% if value.role %}
-            <span>{{ value.role }}</span>
-        {% endif %}
-    </cite>
+                {% if value.company %}
+                    <span class="blockquote__company">{{ value.company }}</span>
```

Comment:
This is a new variable and didn't exist in the previous template. (writing this i'm wondering if it's needed and if we could use role to output both the job title and company?)

Reply:
In the fieldspec I put role and company in the same variable and called it 'role / company'. If the back-end code has called it role we can stick with that, but might want to amend the name the editor sees.

Reply:
It looks like the back-end hasn't been merged for any of these new fields yet - can you ask Lauren what variable names she's using to keep this in sync?

Reply:
Thanks, removed in https://github.com/torchbox/torchbox.com/pull/26/commits/e23dc70728938f8c2e484ecafbcfca1b22fe2879